engines/rancher_on_eks/app/models/rancher_on_eks/cluster_size.rb

```diff
@@ -9,12 +9,26 @@ class ClusterSize
       large: 'm5a.2xlarge'
     }
 
+    TYPES_INDEX = {
+      small: 0,
+      medium: 1,
+      large: 2
+    }
+
     def initialize(*args)
       @size = KeyValue.get('cluster_size', 'small')
     end
 
     def instance_type
-      TYPES_FOR_SIZE[@size.to_sym]
+      @region = AWS::Region.load
+      supported_instance_type = @region.supported_instance_type
+      supported_instance_type = supported_instance_type.split(',')
+      # TYPES_FOR_SIZE[@size.to_sym]
```

Comment:
Leftover code that was commented out for testing?

Reply:
good catch, that was the original code